### PR TITLE
Add user from reacted message as a responder

### PR DESCRIPTION
**Pull request by joshuatobin**
CC: @jmervine @cmeckhardt

### CODE REVIEW

**Comment by jmervine** [comment]
One comment, otherwise looks good.

File: app/controllers/api/v1/chat_ops_controller.rb [hunk 1]
```diff
@@ -49,12 +49,15 @@ def slack_event
...
     # and respond to the user who reacted to the message with a private confirmation
+    user = slack_user_from_message_ts
     current_incident.timeline_entries.create!(
-      user:      slack_user_from_message_ts,
+      user:      user,
       message:   slack_message_from_timestamp["text"],
       timestamp: Time.at(slack_message_from_timestamp["ts"].to_i).utc
     )
 
+    current_incident.responders += [user]
```
Comment (lexelby) [inline app/controllers/api/v1/chat_ops_controller.rb]: Nope, not for a HABTM.  However, I think the idiomatic way of writing this is:

`current_incident.responders << user`

**Comment by joshuatobin** [comment]
@jmervine @lex Added it via `<<`

File: app/controllers/api/v1/chat_ops_controller.rb (hunk 1 above)
Comment (joshuatobin) [inline app/controllers/api/v1/chat_ops_controller.rb]: `has_and_belongs_to_many`

**Comment by lexelby** [comment]
:+1:

**Pull request merged**